orttraining/orttraining/python/training/ortmodule/_runtime_inspector.py

```diff
@@ -146,7 +146,7 @@ def initialize(self, model: ModelProto, user_input_names: List[str]) -> None:
             self._tensor_to_node_map.clear()
             for node in model.graph.node:
                 for output_name in node.output:
-                    if output_name != "":  # noqa: PLC1901
+                    if output_name != "":
```

Comment:
Why is the `noqa` no longer needed? Did the linter do this automatically?

Reply:
yes lintrunner -a helps remove it.

Reply:
That said we should still prefer

if output_name: